server.sy

```diff
@@ -1,10 +1,29 @@
 use game
+use text
 
 main_loop :: fn {
     n_rpc_start_server'
 
+    // Ask for server name
+    server_name := "Puck.NET"
+    l_set_text_input_enabled' true
```

Comment:
We don't need to do this - do we? If we don't we could trigger escape easily

Reply:
As discussed, otherwise the string we pass to `l_text_input_update` won't update.